lms/static/sass/course.scss

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
@ormsbee, any reason why you're deleting these ``course-*.scss`` files? Looks like they're generated by a ``.mako`` template via Stanford theming, but just curious why they're a part of this work.